src/components/HomePage.js

```diff
@@ -18,19 +18,18 @@ import React from 'react'
 
 var linkable_clients = [
     {
-        name: "Riot",
+        name: "Element",
         logo: {
-            width: "48px",
-            src: "img/riot@2x.png",
-            srcSet: "img/riot.png, img/riot@2x.png 2x",
+            width: "64px",
+            src: "img/element-64px.png",
```

Comment:
Everything else looks to be using a 48px logo, won't this make it look out of place?

Reply:
I intentionally left it bigger because the logo's whitespace actually makes it much smaller than the others:

at 64:
![image](https://user-images.githubusercontent.com/23084468/87619434-1abba580-c714-11ea-92b9-0aa74d5077f2.png)

at 48:
![image](https://user-images.githubusercontent.com/23084468/87619452-23ac7700-c714-11ea-9d80-f2e258617f79.png)